Update Google AI to allow HASS access (#32839)
home-assistant/home-assistant.github.io: source/_integrations/google_generative_ai_conversation.markdown

ha_integration_type: service
---

The Google Generative AI integration adds a conversation agent powered by [Google Generative AI](https://ai.google.dev/) in Home Assistant. It can optionally be allowed to control Home Assistant.

> It can only query information that has been provided by Home Assistant in the prompt. To be able to answer questions about your house, Home Assistant will need to provide Google Generative AI with the details of your house, which include areas, devices and their states. This prompt can be customized to adjust the provided information and how the AI should respond.

Reviewer: ```suggestion
It can only query information that has been provided by Home Assistant in the prompt. To be able to answer questions about your home, Home Assistant will need to provide Google Generative AI with the details of your home, which includes areas, devices, and their states. This prompt can be customized to adjust the provided information and how the AI should respond.
```

{% include integrations/option_flow.md %}
{% configuration_basic %}

> Model:

Reviewer 2: Duplicate.

Reviewer 2: Removing the duplicate in https://github.com/home-assistant/home-assistant.io/pull/32857

  description: Model used to generate response.

> Control Home Asssistant:

Reviewer: ```suggestion
Control Home Assistant:
```

  description: If the model is allowed to interact with Home Assistant

Model:
  description: Model used to generate response.

> Prompt Template:

Reviewer: ```suggestion
Prompt template:
```
